scripts/quality/normalize_lcov.py

```
@@ -0,0 +1,69 @@
+    parser = argparse.ArgumentParser(
+        description="Normalize LCOV output so downstream tools measure line coverage consistently."
+    )
+    parser.add_argument("--input", required=True, help="Path to the input lcov.info file")
+    parser.add_argument("--output", required=True, help="Path to the normalized output lcov.info file")
+    return parser.parse_args()
+
+
+def _resolve_repo_path(raw_path: str, *, label: str, must_exist: bool) -> Path:
+    candidate = Path((raw_path or "").strip())
+    if not str(candidate):
+        raise SystemExit(f"{label} path is required")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)